Sources/VLCMedia.m

```diff
@@ -1015,21 +1036,11 @@ - (void)setStateAsNumber:(NSNumber *)newStateAsNumber
     [self setState: [newStateAsNumber intValue]];
 }
 
-#if TARGET_OS_IPHONE
```

Comment:
Why were there two separate `metaDictionary` functions for iOS and non-iOS platforms?